doc/manual/installation/prerequisites-source.xml

```diff
@@ -52,6 +58,10 @@
   pass the flag <option>--enable-gc</option> to
   <command>configure</command>.</para></listitem>
 
+  <listitem><para>The <literal>boost</literal> library of version
+  1.61.0 or higher. It can be obtained from the official web site
```

Comment:
I believe it should be higher due to some boost::coroutine2 bug in previous releases. Maybe 1.65 but I'm not sure.

Reply:
@edolstra
I guess you are referring to
https://github.com/NixOS/nix/blob/03ce0c3a9e1c07043445579c619eed4895647b36/src/libutil/serialise.cc#L160-L162
It singles out 1.63.x, 1.64.x, 1.65.x to exclude specifically. A test can be done a following way:
```
with import <nixpkgs> {};

map (boost: nix.override { inherit boost; })
[
  #boost155
  #boost159
  #boost160 ld: symbol(s) not found for architecture x86_64
  #boost161 not in nixpkgs, but works if you add it
  boost162
  #boost163 assert fails in serialise.cc
  #boost164 assert fails in serialise.cc
  #boost165 assert fails in serialise.cc
  boost166
  boost167
  boost168
]
```
All four build for me (including `installCheckPhase`) just fine. I can change the minimum version to 1.66, if you think that is needed.

Reply:
Thanks for investigating :-)